src/models/onnxruntime_api.h

```diff
@@ -175,6 +179,12 @@ inline void InitApi() {
     return;
   }
 
+  const std::string ort_lib = Generators::GetEnvironmentVariable("ORTGENAI_LOG_ORT_LIB");
+  if (ort_lib == "1" || ort_lib == "true") {
```

Comment:
Do we have similar checks elsewhere in the project? Maybe we need a consolidated way to check for `true` and `false`.

Reply:
Will make a utility for this.